src/bin/rustfmt.rs

```diff
@@ -112,6 +113,14 @@ fn match_cli_path_or_file(config_path: Option<PathBuf>,
 fn update_config(config: &mut Config, matches: &Matches) {
     config.verbose = matches.opt_present("verbose");
     config.skip_children = matches.opt_present("skip-children");
+
+    let write_mode = matches.opt_str("write-mode").map(|ref s| {
+        WriteMode::from_str(s).expect(&format!("Invalid write-mode: {}", s))
```

Comment:
We shouldn't panic due to invalid user input. We should exit gracefully with the message.